### PR TITLE
Adds .project and .settings to .gitignore

### PR DESCRIPTION
**Pull request by vogella**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,6 @@
 local.properties
 build
 .gradle
+.project
+.settings
```
Comment (nkoroste) [inline .gitignore]: shouldn't this be ".settings/" ?

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,6 @@
 local.properties
 build
 .gradle
+.project
```
Comment (nkoroste) [inline .gitignore]: I would add the following comment as well, "# Eclipse project files" also probably include ".classpath" as well

**Comment by nkoroste** [comment]
Since Gradle files can be used to import the project I think its okay to add those files to .gitignore even though the official Eclipse documentation advise against it:
http://wiki.eclipse.org/FAQ_How_do_I_set_up_a_Java_project_to_share_in_a_repository%3F

**Comment by vogella** [comment]
What is the right way to update a Github PR based on your comment? Amend the existing commit or simply add another commit?

**Comment by vogella** [comment]
Commit was amended based on your comments.

**Comment by otw** [comment]
LGTM

**Pull request merged**